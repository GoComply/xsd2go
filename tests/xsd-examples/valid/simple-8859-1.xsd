<?xml version="1.0" encoding="ISO-8859-1"?>
<xsd:schema	xmlns:xsd="http://www.w3.org/2001/XMLSchema"
		xmlns:simple-schema="https://simple.example.com/"
		targetNamespace="https://simple.example.com/"
		elementFormDefault="qualified">
	<xsd:element name="myelement" type="simple-schema:MyElementType" />
	<xsd:complexType name="MyElementType">
		<xsd:sequence>
			<xsd:element name="id" type="xsd:integer" />
		</xsd:sequence>
		<xs:annotation>
			<xs:documentation>Documentation with a character from ISO-8859-1 encoding: Ã±</xs:documentation>
		</xs:annotation>
	</xsd:complexType>
</xsd:schema>
