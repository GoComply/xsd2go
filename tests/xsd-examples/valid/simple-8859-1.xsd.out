// Code generated by https://github.com/gocomply/xsd2go; DO NOT EDIT.
// Models for https://simple.example.com/
package simple_schema

import (
	"encoding/xml"
)

// Element
type Myelement struct {
	XMLName xml.Name `xml:"myelement"`

	Id int64 `xml:",any"`
}

// XSD ComplexType declarations

type MyElementType struct {
	XMLName xml.Name

	Id int64 `xml:",any"`

	InnerXml string `xml:",innerxml"`
}

// XSD SimpleType declarations
